### PR TITLE
xmrig_ir

### PR DESCRIPTION
**Pull request by tccontre**
PR for CPU Miner Found in ATTACK_RANGE - HoneyPOT:

## New XMRIG analytics stories

XMRig

## New detections:

Deleting Of Net Users
Disable Windows App Hotkeys
Disabling Net User Account
Download Files Using Telegram
Enumerate Users Local Group Using Telegram
Excessive Attempt To Disable Services
Excessive Service Stop Attempt
Excessive Usage Of Cacls App
Excessive Usage Of Net App
Excessive Usage Of Taskkill
Executables Or Script Creation In Suspicious Path
Hide User Account From Sign-In Screen
Icacls Deny Command
ICACLS Grant Command
Modify ACL permission To Files Or Folder
Process Kill Base On File Path
Schtasks Run Task On Demand
Suspicious Driver Loaded Path
Suspicious Process File Path
XMRIG Driver Loaded

## minor description mod:

Schedule Task with Rundll32 Command Trigger

### CODE REVIEW

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Excessive Deleting Of Users :white_check_mark:
- diskUsage: 503808
- runDuration: 1.69s
- scanCount: 326

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Disable Windows App Hotkeys :white_check_mark:
- diskUsage: 389120
- runDuration: 2.019s
- scanCount: 1861

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Disabling Net User Account :white_check_mark:
- diskUsage: 286720
- runDuration: 1.743s
- scanCount: 72

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Download Files Using Telegram :white_check_mark:
- diskUsage: 294912
- runDuration: 1.122s
- scanCount: 44

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Enumerate Users Local Group Using Telegram :white_check_mark:
- diskUsage: 159744
- runDuration: 0.68s
- scanCount: 1

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Excessive Attempt To Disable Services :white_check_mark:
- diskUsage: 405504
- runDuration: 129.474s
- scanCount: 271458

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Excessive Service Stop Attempt :white_check_mark:
- diskUsage: 413696
- runDuration: 127.265s
- scanCount: 271458

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Excessive Usage Of Cacls App :white_check_mark:
- diskUsage: 417792
- runDuration: 133.112s
- scanCount: 271458

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Excessive Usage Of Net App :white_check_mark:
- diskUsage: 413696
- runDuration: 129.423s
- scanCount: 271458

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Executables Or Script Creation In Suspicious Path :white_check_mark:
- diskUsage: 544768
- runDuration: 3.454s
- scanCount: 2907

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Excessive Usage Of Taskkill :white_check_mark:
- diskUsage: 409600
- runDuration: 134.358s
- scanCount: 271458

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Hide User Account From Sign-In Screen :white_check_mark:
- diskUsage: 385024
- runDuration: 2.5540000000000003s
- scanCount: 1861

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Icacls Deny Command :white_check_mark:
- diskUsage: 503808
- runDuration: 1.481s
- scanCount: 84

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

ICACLS Grant Command :white_check_mark:
- diskUsage: 503808
- runDuration: 36.865s
- scanCount: 72969

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Modify ACL permission To Files Or Folder :white_check_mark:
- diskUsage: 286720
- runDuration: 1.984s
- scanCount: 49

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Schtasks Run Task On Demand :white_check_mark:
- diskUsage: 495616
- runDuration: 44.943s
- scanCount: 97640

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Process Kill Base On File Path :white_check_mark:
- diskUsage: 409600
- runDuration: 133.501s
- scanCount: 271458

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Suspicious Driver Loaded Path :white_check_mark:
- diskUsage: 139264
- runDuration: 0.711s
- scanCount: 40

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

XMRIG Driver Loaded :white_check_mark:
- diskUsage: 135168
- runDuration: 1.094s
- scanCount: 59

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Suspicious Process File Path :white_check_mark:
- diskUsage: 569344
- runDuration: 2.7880000000000003s
- scanCount: 2830

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Ransomware Notes bulk creation :white_check_mark:
- diskUsage: 196608
- runDuration: 1.587s
- scanCount: 524

File: detections/endpoint/executables_or_script_creation_in_suspicious_path.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+author: Teoderick Contreras, Splunk
+type: batch
+datamodel:
+- Endpoint
+description: This search is to detect a suspicious executable or scripts (known file
+  extensions) in list of suspicious file path in windows OS. This technique was seen
+  in so many adversaries and attacker to hide their track from the user. The suspicious
+  file path in detection are known dropping point uses in the wild and folder path
+  that are not commonly to have executable or scripts.
+search: "|tstats `security_content_summariesonly` values(Filesystem.file_path) as\
```
Comment (P4T12ICK) [inline detections/endpoint/executables_or_script_creation_in_suspicious_path.yml:14]: I think we have too many backslashes in here?

Comment (P4T12ICK) [inline detections/endpoint/executables_or_script_creation_in_suspicious_path.yml]: here are many backslashes?

**Review by P4T12ICK — CHANGES_REQUESTED** [review]

**Review by tccontre — COMMENTED** [review]
already re copy the original detection I have. thanks for noticing this one

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Executables Or Script Creation In Suspicious Path :white_check_mark:
- diskUsage: 540672
- runDuration: 3.8360000000000003s
- scanCount: 2898

**Review by P4T12ICK — DISMISSED** [review]

**Review by MHaggis — DISMISSED** [review]
Everything looks good. I'd maybe add a bit more data to the analytic story to provide some more context. Otherwise, good to go! :shipit:

**Comment by tccontre** [comment]
> Everything looks good. I'd maybe add a bit more data to the analytic story to provide some more context. Otherwise, good to go! :shipit:

thanks guys for this PR review and updates. :)

**Review by MHaggis — APPROVED** [review]

**Review by patel-bhavin — APPROVED** [review]

**Pull request merged**